### DISCUSSION TITLE
FNIRSI GC-01: Recover bricked device

### DISCUSSION THREAD

**Discussion by davidkirr** [Gissio/radpro#139] [Q&A] [answered]
Please help. By mistake uploaded CH32 firmware to a device with APM32 chip. The device is now bricked. How can I unbrick it please? When I connect to USB it gives me ERRAPP.TXT or UNKNOWN.TXT When I try to copy the stock firmware nothing happens. Tried the reset button on device as well. Nothing. Please please help. Thanks

**Comment by alfmck** [comment, marked as answer]
Strange. ARM32 is more powerful chip and successfully works with CH32 and APM32 firmware. You should use PC with win10 or win11.

**Reply by davidkirr** [reply]
Thank you so much for this. It helped. I was trying to install it via Linux Mint and then via Windows 7 neither of them worked. Windows 11 has done the job. Thanks again
